### PR TITLE
GET candidate routes

### PR DESCRIPTION
**Pull request by syncush**
added - candidates information getters routes.
fixed a bug - a typo in vision package caused server fail to load
added - package.json added myself as a contributor

### CODE REVIEW

File: packages/server/src/index.js [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
     info: {
         title: 'Candidate management API',
         version: Pack.version,
-    }
+    },
 };
 
 const server = Hapi.server({
-    port: 80 || process.env.PORT,
+    port: process.env.PORT || 80,
```
Comment (dormesica) [inline packages/server/src/index.js]: Haha, can't believe I missed that

File: packages/server/src/index.js [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
 });
 
 server.route(require('./api/isAlive'));
+server.route(require('./api/candidate/getByID'));
```
Comment (dormesica) [inline packages/server/src/index.js]: Unify these under a single export such that adding it to the server would take only 1 line. I.e.
```javascript
server.route(require('./api/candidate');
```

These route configuration will grow. We wouldn't want to overwhelm the file with a line for each path.
I would be much cleaner to once add all the paths relevant to candidates, then reviewer etc.

File: package.json [hunk 3]
```diff
@@ -7,13 +7,17 @@
...
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "run-server-nodemon": "PORT=3000 nodemon packages/server/src/index.js"
     },
     "repository": {
         "type": "git",
         "url": "git+https://github.com/dormesica/candidate.git"
     },
     "author": "Dor Mesica <dormesica10@gmail.com>",
+    "contributors": [
+        "Daniel Hermon <danielhermonsync@gmail.com"
```
Comment (dormesica) [inline package.json]: Close your mail tag.

File: packages/server/src/api/candidate/getByID.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+                id: Joi.string()
+                    .regex(/[0-9]+/)
+                    .length(9)
+                    .trim(true)
+                    .truncate(true)
+                    .required(),
+            },
+        },
+        response: {
+            schema: candidateSchema,
```
Comment (dormesica) [inline packages/server/src/api/candidate/getByID.js]: I think I might have seen somewhere that it is acceptable to add a link to the next page... Might bt worth checking

File: packages/server/src/api/candidate/schemas.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+const Joi = require('joi');
+module.exports = Joi.object({
```
Comment (dormesica) [inline packages/server/src/api/candidate/schemas.js]: Keep in mind that this file sould probably include more schemas to be exported so exporting a single validation scheme might require more changes in the future.

Or was it intentional?

File: packages/server/src/api/candidate/schemas.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+const Joi = require('joi');
```
Comment (dormesica) [inline packages/server/src/api/candidate/schemas.js]: Give a space line between `require` and code please.

File: packages/server/src/api/candidate/schemas.js (hunk 5 above)
Comment (syncush) [reply packages/server/src/api/candidate/schemas.js]: At first, I wanted to include the schemas for input, output validation but at second thought I thought it was overkill,  so I thought it is better to write the schema for the resource which in this case is the candidate and use it properly in the getAll & getByID route object.

**Pull request merged**